cmake/projects/Boost/schemes/url_sha1_boost.cmake.in

```diff
@@ -154,6 +154,7 @@ ExternalProject_Add(
     install
     ${boost_list}
     "--prefix=@HUNTER_PACKAGE_INSTALL_PREFIX@"
+    --ignore-site-config
```

Comment:
Have you tried to run this code with other components like boost.system? We don't need `--ignore-site-config` in `url_sha1_boost_library.cmake.in`?